Extensions/view_on_dash.js

```diff
@@ -48,14 +48,12 @@ XKit.extensions.view_on_dash = new Object({
 
 		if (this.preferences.show_sidebar_button.value === true) {
 
-			if ($(".account_header").length > 0) {
-				xf_html = '<ul class="controls_section" id="view_on_dash_ul">' +
-					'<li class="section_header selected">VIEW BLOGS</li>' +
-					'<li class="no_push"><a href="#" onclick="return false;" id="view_on_dash_button">' +
-						'<div class="hide_overflow">View on Dash<span class="sub_control link_arrow arrow_right"></span></div>' +
-					'</a></li></ul>';
-				$("ul.controls_section:eq(1)").before(xf_html);
-			}
+			xf_html = '<ul class="controls_section" id="view_on_dash_ul">' +
```

Comment:
Looks like this leaks the xf_html variable to the global scope. I don't see anywhere where xf_html is (or was) declared.